src/platform/common/utils/multiStepInput.ts

```diff
@@ -206,7 +206,7 @@ export class MultiStepInput<S> implements IMultiStepInput<S> {
                                     input.enabled = false;
                                     input.busy = true;
                                     // No validation allowed on a quick pick. Have to put up a dialog instead
-                                    await this.shell.showErrorMessage(validationMessage, { modal: true });
+                                    await this.shell.showErrorMessage(validationMessage);
```

Comment:
This fixes the new experimental flow. It was using a quick pick instead of an input box. There error message that it was showing was being forced to modal. But at this point the flow just ends regardless of modal or not. So there is no need to make this a modal dialog that can't show links. Just use a normal error message. This is the only multistep quick pick flow that uses validation, so we know it won't change any other flows.